docs/source/basics/adding_secrets.md

```diff
@@ -0,0 +1,55 @@
+# Storing and using Secrets
+
+Often a workflow can depend on _secret data_, such as an API key, to function
+correctly. To make storing and using secret data easy, the Latch SDK comes with
+special utilities that handle this securely.
+
+## Adding Secrets
+
+To add a secret, simply navigate to the [Latch Console](https://console.latch.bio)
+and head to Account Settings > Developer Settings.
+
+![How to get to Account Settings](../assets/settings-menu.png)
+
+![How to get to Developer Settings](../assets/developer-settings.png)
+
+From there, scroll down to the 'Secrets' section and add your secrets. Secrets
+consist of key-value pairs where keys are unique across a workspace. Secrets are
+also immutable, so the only way to change the value of a given secret is to
+delete it and add a new one with the same key.
+
+## Using Secrets
+
+To use a secret in a workflow, simply use the `get_secret` function built into
+the Latch SDK. This function takes in a key and returns the value of the secret
```

Comment:
```suggestion
the Latch SDK. This function takes in a key, which is the name of the secret created on Latch Console, and returns the value of the secret
```

Reply:
i think this is unnecessary because its explained above that keys identify secrets and nowhere else is the word 'name' used

Reply:
don't feel strongly about it. feel free to merge